src/book/mod.rs

```diff
@@ -244,8 +244,9 @@ impl MDBook {
         self
     }
 
-    /// Run `rustdoc` tests on the book, linking against the provided libraries.
-    pub fn test(&mut self, library_paths: Vec<&str>) -> Result<()> {
+    /// Run `rustdoc` tests on the book, linking against the provided libraries
+    /// and optionally testing only a signal named chapter while skipping the others.
+    pub fn test(&mut self, library_paths: Vec<&str>, chapter: Option<&str>) -> Result<()> {
```

Comment:
Unfortunately we can't change the API as that would be a semver breaking change.

I think you will need to add a new method, and have the old `test` call it.

Reply:
done, I added "test_chapter", but instead of duplicating the code I just make test call test_chapter.